tests/_support/MockCustomer.php

```diff
@@ -34,7 +34,10 @@
  *
  */
 
+namespace Nosto\Test\Support;
+
 use Nosto\Object\Customer;
+use DateTime;
```

Comment:
Are you sure this is correct? Shuold it be `\DateTime`. Without the leading backslash we're trying to find `DateTime` class from the same namespace (`Nosto\Test\Support`)

Reply:
It does recognise the right class